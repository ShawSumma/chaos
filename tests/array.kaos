array a = [1, 2, 3]
print a
list b = [true, false]
print b
list c = [3.2, 345.1665]
print c
list d = ['a', 'b', 'c']
print d
array e = ["A", "B", "C"]
print e
list f = ['A', "asdasdad12312", 1232, 435.16, true, false]
print f
list g = ["asdasdasd,", 123123]
print g

bool list arr1 = [true, false, true]
print arr1
number list arr2 = [1, 2, 63.3, 12321.1515]
print arr2
string array arr3 = ['A', "asdasdaqs", 'asdasd123123', "."]
print arr3
