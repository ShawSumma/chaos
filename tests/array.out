[1, 2, 3]
[true, false]
[3.2, 345.167]
['a', 'b', 'c']
['A', 'B', 'C']
['A', 'asdasdad12312', 1232, 435.16, true, false]
['asdasdasd,', 123123]
[true, false, true]
[1, 2, 63.3, 12321.2]
['A', 'asdasdaqs', 'asdasd123123', '.']
[start] =>
	{name: a, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: b, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: c, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: d, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: e, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: f, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: g, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: arr1, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 0, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: arr2, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 1, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: arr3, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 2, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
[end]
[start] =>
	{name: a, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: b, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: c, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: d, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: e, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: g, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 3, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: arr1, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 0, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{name: arr2, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 1, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
	{name: arr3, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 4, 2nd_type: 2, value_type: 4, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: (null), 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
[end]
3
2
1
3
2
1
false
345.167
c
A
63.3
.
[5, 2, 3]
32
[5, 2, 3]
[1, 2, 3]
[1, 2, 3]
[5, 2, 3]
[1, 2, 3]
[5, 2, 3]
[false, false]
[5, 2, 3]
['d', 'b', 'c']
[1, 2, 3, 4, 5]
[1, 3, 4, 5]
[1, 3, 5]
[3, 5]
[3]
[]
1
2
3
2
true
true
1
3
a
c
true
true
1
3
a
c
true
true
3
1
c
a
true
true
3
1
c
a
[false, false, true]
[false, false, false]
[5, 2, 3]
[5, 2, 6]
['d', 'b', 'c']
['d', 'b', 'e']
[false, false]
